### PR TITLE
feat:implement cards UI

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@hungnguyenhuu-agilityio @lenguyenthimy-agilityio Please help me to check and merge it. I use some url for example my UI at src property of img tag!

### CODE REVIEW

File: src/home.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          </li>
+          <li class="text text--2xl language__item">
+            China
+            <img src="assets/icons/delete.svg" alt="icon-delete" class="language__delete" />
+          </li>
+        </ul>
+      </section>
+      <!-- end language        -->
+
+      <!-- start cards         -->
```
Comment (hungnguyenhuu-agilityio) [inline src/home.html]: You should leave TODO here to note this section is template render. You need to update

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated it. Please help me to check again

**Comment by hungnguyenhuu-agilityio** [comment]
Conflict!

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated it. Please help me to check again

File: src/home.html [hunk 2]
```diff
@@ -34,9 +34,45 @@ <h1 class="text text--light text--lg text--extrabold text--uppercase">Flashcards
             <img src="assets/icons/delete.svg" alt="icon-delete" class="language__delete" />
           </li>
         </ul>
+        <!-- this section will be render by template -->
+        //TODO: this section will be render by template
```
Comment (hungnguyenhuu-agilityio) [inline src/home.html]: TODO should be a comment.

File: src/styles/pages/home/cards.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.card:hover {
+  opacity: 0.8;
+}
+
+.card__term {
+  padding: 30px 0 10px 0;
+}
+.card__picture {
```
Comment (hungnguyenhuu-agilityio) [inline src/styles/pages/home/cards.css:49]: should we break the line before?

File: src/styles/pages/home/cards.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+.cards__add:hover {
+  opacity: 0.9;
+}
+
+.card__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 90px;
+}
+.card {
```
Comment (hungnguyenhuu-agilityio) [inline src/styles/pages/home/cards.css:29]: break-line here?

File: src/home.html (hunk 2 above)
Comment (LeVHoangduc) [reply src/home.html]: @hungnguyenhuu-agilityio I have updated it.

File: src/styles/pages/home/cards.css (hunk 3 above)
Comment (LeVHoangduc) [reply src/styles/pages/home/cards.css:49]: @hungnguyenhuu-agilityio  I have updated it.

File: src/styles/pages/home/cards.css (hunk 4 above)
Comment (LeVHoangduc) [reply src/styles/pages/home/cards.css:29]: @hungnguyenhuu-agilityio  I have updated it.

File: src/home.html (hunk 2 above)
Comment (hungnguyenhuu-agilityio) [reply src/home.html]: //TODO: this section will be render by template

this one is not the comment of HTML file, right?

Comment (LeVHoangduc) [reply src/home.html]: @hungnguyenhuu-agilityio I have updated it. Please help me to check again.

**Comment by hungnguyenhuu-agilityio** [comment]
Conflict!

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have resolved the conflict.

**Pull request merged**